features/step_definitions/pv.rb

```diff
@@ -156,22 +156,21 @@
     eval "pv_hash#{path} = YAML.load value" unless path == ''
     # e.g. pv_hash["spec"]["nfs"]["server"] = 10.10.10.10
   end
+  pv_name = pv_hash["metadata"]["name"]
 
   logger.info("Creating PV:\n#{pv_hash.to_yaml}")
   @result = BushSlicer::PersistentVolume.create(by: admin, spec: pv_hash)
 
   if @result[:success]
-    cache_resources *@result[:resource]
-
-    # register mandatory clean-up
-    _pv = @result[:resource]
     _admin = admin
     teardown_add {
-      @result = _pv.delete(by: _admin)
```

Comment:
Sorry, I met this error: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/40227/console, and not sure why. Does `delete_deps` here work?

Reply:
@weherdh Fixed. See Runner-v3/40240
Also please update your comments to remove the internal URL.